joomla-projects/gsoc21_media-manager
# Week 9: Final fixes and improvements

Pull Request for issue #23.

### Summary of Changes
- Added final fixes and improvements to responsive images functionality.
- Implemented functions for comparing form and content images and detecting responsive versions that are no longer used.


### Testing Instructions
Go to an article, insert an image using content editor or form, play with sizes & creation method and observe the front-end - images will be generated accordingly. Additionally, a browser alert containing the unused images will pop up on save:

![image](https://user-images.githubusercontent.com/62054743/130178581-7df6983b-f511-4b94-99bd-6b85dfcad45b.png)
